controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/security/KeyServiceMock.java

```diff
@@ -0,0 +1,13 @@
+package com.yahoo.vespa.hosted.controller.api.integration.security;
+
+/**
+ * @author mpolden
+ */
+public class KeyServiceMock implements KeyService {
```

Comment:
Note: We already have an interface for secret providers in `jdisc_http_service` ( `SecretStore`), with implementation residing `jdisc_http_filters` in _vespa-yahoo_ ( `YkeykeySecretStore` ).

Reply:
We are currently using `SecretStore` in _athenz-instance-provider-service_ running on the config servers.